kubeadm/v1.15.0/KubeCluster.ps1

```diff
@@ -0,0 +1,273 @@
+Param(
+    [parameter(Mandatory = $false,HelpMessage="Print the help")]
+    [switch] $help,
+    [parameter(Mandatory = $false,HelpMessage="Install pre-requisites")]
+    [switch] $install,
+    [parameter(Mandatory = $false,HelpMessage= "Join the windows node to the cluster")]
+    [switch] $join,
+    [parameter(Mandatory = $false,HelpMessage="Reset and clean up this Windows worker node")]
+    [switch] $reset,
+    [parameter(Mandatory = $false,HelpMessage="Path to input configuration JSON")] 
+    $ConfigFile
+)
+
+# Handle --help
+if ($help.IsPresent)
+{
+    Usage
```

Comment:
this needs to go below Usage() otherwise `./kubecluster.ps1 -help` throws an error saying Usage is not defined